### PR TITLE
OPS: Use new Image Picker

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Review by Overtorment — CHANGES_REQUESTED** [review]
looks like unit tests fail because old `react-native-image-picker` mock is no good for the new one

File: __mocks__/react-native-image-picker.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {NativeModules} from 'react-native';
+
+// Mock the ImagePickerManager native module to allow us to unit test the JavaScript code
```
Comment (Overtorment) [inline __mocks__/react-native-image-picker.js:3]: capt. obvious comments kek

File: __mocks__/react-native-image-picker.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+NativeModules.ImagePickerManager = {
+  showImagePicker: jest.fn(),
+  launchCamera: jest.fn(),
+  launchImageLibrary: jest.fn(),
+};
+
+// Reset the mocks before each test
+global.beforeEach(() => {
```
Comment (Overtorment) [inline __mocks__/react-native-image-picker.js:11]: I think this belongs in `tests/setup.js ` but ok

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4ue458eh4vax8ay2hwg00z8y68

[download apk](http://filestorage.bluewallet.io:1488/1607643749.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**